content/sensu-go/6.2/observability-pipeline/observe-schedule/collect-metrics-with-checks.md

```diff
@@ -34,40 +34,6 @@ sensuctl check set-output-metric-format collect-metrics nagios_perfdata
 sensuctl check set-output-metric-handlers collect-metrics prometheus_gateway
 {{< /code >}}
 
```

Comment:
We should update this guide to move more of the reference-type information (stuff like "you can XYZ") into the metrics reference. We could also repurpose it to walk through using one of the curated templates for metrics, like InfluxDB (maybe?)